hikari/api/shard.py

```diff
@@ -392,3 +257,6 @@ async def request_guild_members(
             When trying to request presences without the `GUILD_MEMBERS` or when trying to
             request the full list of members without `GUILD_PRESENCES`.
         """
+
+    def __await__(self) -> typing.Generator[None, typing.Any, typing.Any]:
```

Comment:
Might replace this with explicit calls to join. This was me being partially lazy, as it is easier to do `asyncio.gather(*bot.shards.values())` than `asyncio.gather(*(s.join() for s in bot.shards.values()))`